### ISSUE TITLE
Pretrained weight

### ISSUE THREAD

**Issue by ldtho** [MingkunLishigure/MaskCL#2]
Hi @MingkunLishigure,
Thank you for the great work. May I know if you'll provide pretrained weights anytime soon? I'm really looking forward to it.
Regards

**Comment by MingkunLishigure** [comment]
Yes, thank you for your attention! We are actively working on further
improvements for this project, and we will release the updated code and
pre-trained weights simultaneously in the future.

Duy Tho Le ***@***.***> 于2023年9月29日周五 19:44写道：
